lib/tcpip/tcpip_error_handler.h

```diff
@@ -31,24 +31,20 @@ extern struct FlagLockState ipThreadLockState;
 /**
  * Other locks and futexes to release as part of the reset. These are
  * documented in `sdk/include/FreeRTOS-Compat/task.h` in the main tree
- * (`__CriticalSectionFlagLock` and `__SuspendFlagLock`) and
- * `source/FreeRTOS_IP.c` in the FreeRTOS+TCP tree (`xNetworkEventQueue`).
+ * (`__CriticalSectionFlagLock` and `__SuspendFlagLock`),
+ * `source/FreeRTOS_IP.c` in the FreeRTOS+TCP tree (`xNetworkEventQueue`), and
+ * `network_wrapper.cc` in the TCP/IP wrapper.
  */
 extern struct RecursiveMutexState __CriticalSectionFlagLock;
 extern struct RecursiveMutexState __SuspendFlagLock;
```

Comment:
Don't these get declared in FreeRTOS.h?

Reply:
I think that the doc comment is correct, they are documented in https://github.com/CHERIoT-Platform/cheriot-rtos/blob/main/sdk/include/FreeRTOS-Compat/task.h#L85

Or am I missing something?